pkgs/tools/typesetting/asciidoc/default.nix

```diff
@@ -158,6 +160,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ python3 unzip autoreconfHook ];
+  buildInputs = [ python3 ];
```

Comment:
```suggestion
  buildInputs = lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [ python3 ];
```

Reply:
I've enabled strictDeps, and it's needed in both nativeBuildInputs and buildInputs. So let's leave it like it is.